script/Ciel/Day 7/07_01_CIEL07_2_1A.txt

```diff
@@ -78,60 +80,62 @@ Senpai doesn't seem to appreciate what happened between me and Noel.
 -- Page 176, Offset 15226.
 -- ……そうですね。座ってもぜんぜん落ち着かないのは疑問があるからです。いい機会なのではっきりさせましょう」
 How do I put it... I guess what mystifies me is that even though you sat down I'm still bothered.
- We might as well take the opportunity to clear something up for me."
+ I suppose I may as well take this opportunity to clear something up."
 }
 [sha:c97365d70967fbd82d50e9ec9f4e236c827d3ac6]{
 -- Page 177, Offset 15227.
 -- シエル先輩はやや前屈みになって俺の顔を上目遣いに覗きこむ。
-Ciel bends forward slightly, peering into my face from below.
+Ciel leans forward slightly, and peers up at my face from below.
 }
 [sha:509906bb152218d63bb8c5ae49ac52917f7e450a]{
 -- Page 178, Offset 15228.
 -- 「自分から呼んでおいて今更ですけど。
 // kinda long, because of 今更
-"You might feel its a bit late asking this after I told you to come here,
+"This may sound like a strange thing to ask at this point, seeing as how I told you to come here, but―――
 }
 [sha:2a0b98697042fdf9db53bb76de5a668990c79014]{
 -- Page 178, Offset 15229.
 -- なぜノエル先生ではなく、わたしに構うんですか？」
-but why are so concerned with me, rather than Noel-sensei?"
+Why is it that you want to spend time with me, rather than Noel-sensei?"
 }
 [sha:3e6fd19b544f7ef9ddf263309425c423af1e4aac]{
 -- Page 178, Offset 15230.
 -- 「？　なぜって、先輩のことが好きだからですけど」
-"Huh? Why, well because I like you, that's why."
+"Why? Well, because I like you, that's why."
 }
 [sha:72b02553a413a8418fb636b127dc9599348fd351]{
 -- Page 179, Offset 15231.
 -- ホント、なに今更な事を訊いてくるんだこの人は？
 // very liberal, trying to tie back to Offset 15228.
-Yeah, that does feel like she's asking a bit too late.
+She's right, that really does feel like a strange thing to ask at this point.
 }
 [sha:f88ac1963bd337c6dc143706187186b2bf1c1734]{
 -- Page 180, Offset 15232.
 -- 「そ―――それは、えっと……先輩として尊敬している、とか、そういう意味で、ですか？」
-"Y-you mean, uhm, you respect me as your Senpai, or something, right?"
+"Y-You mean, um, that you respect me as your Senpai, or something, right?"
 }
 [sha:b22fb49fbc6db55b602a55bc405362a38775826b]{
 -- Page 180, Offset 15233.
 -- 「それもありますけど、女の子として好きです。チャンスがあれば先輩と交際したいぐらい」
-"Well, I do that too, but I meant I like you as a woman――enough so that I'd wanna go out with you if I got the chance."
+"Well, yeah, that too, but I meant I like you as a woman.
+ So much so that I'd go out with you if given the chance."
 }
 [sha:eb890701590d706cad5b9ce220da0053eb32ec5a]{
 -- Page 181, Offset 15234.
 -- 「チャ、チャンスがあれば、ですか。それは、はい、完全否定はしません、けど」
-"I-if you get the chance, is it? That's, right, not entirely out of the picture, I guess."
+"I-If given the chance, you say?
+ That's... well... not entirely out of the picture, I guess."
```

Comment:
I feel like 'right' works fine as an interjection here.

Replacing so many interjections with 'well' gives a homogenous quality to the dialogue.

Reply:
Disagree with 'right', but I can make it 'yeah' or something

Reply:
Sure